nessie/billRequests.py

```diff
@@ -1,58 +1,56 @@
-import json
 import requests
 
-from nessie.models.bill import Bill
+from .models.bill import Bill
+from .utils.exceptions import NessieApiError
 
 class BillRequest():
 
     # perhaps need better way of injecting key?
     def __init__(self, key):
         self.base_url = "http://api.reimaginebanking.com"
         self.key = key
-        pass
-    
-    def get_account_bills(self, account_id):
-        url = f'{self.base_url}/accounts/{account_id}/bills/?key={self.key}'
-        raise NotImplementedError
-        pass
-
 
     # Get the bill using the bill_id
     # will return
-    # TODO: better way of error handling
-    # TODO: better way of appending id
+    # TODA: better way of error handling
```

Comment:
TODA lol

Reply:
TODAAA

Reply:
the dumb pylint yells if i put a TODO statement